plugins/rbac/dev/index.tsx

```diff
@@ -48,22 +48,44 @@ class MockRBACApi implements RBACAPI {
     return mockPolicies;
   }
 
+  async getAssociatedPolicies(
+    entityReference: string,
+  ): Promise<RoleBasedPolicy[]> {
+    return (
+      mockPolicies.filter(pol => pol.entityReference === entityReference) ||
```

Comment:
Doesn't the `filter` function return an empty array if there's no matches?  I think then the `||` here isn't needed.